spec/controllers/adopters_controller_spec.rb

```diff
@@ -0,0 +1,180 @@
+require 'spec_helper'
+
+describe Api::V1::AdoptersController do
+  
+  let!(:user) { create(:user, account_active: 'true', permissions: 'adopters_edit') }
```

Comment:
Yo cambiaría el `let!` por `let` ya que los user nunca cambian, creo que no es necesario crear uno para cada it